experimental/PropertyDDS/packages/property-properties/src/properties/baseProperty.js

```diff
@@ -1225,15 +1180,26 @@ BaseProperty.prototype._validateInsertIn = function (in_targetParent, in_childId
         }
         parent = parent._parent;
     }
-    // Outside the checked out paths?
-    const repositoryInfo = in_targetParent._getCheckedOutRepositoryInfo();
-    if (repositoryInfo && !repositoryInfo._allCoveredByCheckout(in_targetParent.getAbsolutePath(), in_childId, this)) {
-        throw new Error(MSG.INSERTED_OUTSIDE_PATHS);
-    }
+
+    return (this._parent === undefined && this._getCheckoutView() === undefined);
 };
 
+/**
+ * Returns the checkedOutRepositoryInfo.
+ * @return {property-properties.CheckoutView~CheckedOutRepositoryInfo} The checkedOut repository info.
+ * @protected
+ */
+BaseProperty.prototype._getCheckedOutRepositoryInfo = function () {
+    if (!this._parent) {
+        return this._checkedOutRepositoryInfo;
+    } else {
+        return this.getRoot() ? this.getRoot()._getCheckedOutRepositoryInfo() : undefined;
+    }
+};
 
 /**
+ * TODO: Remove it later. Kept not to modify tests
```

Comment:
Did you add this line ? If not, maybe its good time to modify the tests.

Reply:
Will clean later when converting to TS